README.md

```diff
@@ -25,10 +17,22 @@ cp .env.example .env
 cp wrangler.toml.example wrangler.toml
 ```
 
-3. Deploy your worker to a site with wrangler
+3. Edit environment variables in `.env` and `wrangler.toml` files
+
+3. Run your worker
 
 ```
-wrangler publish --env production
+npm run dev
 ```
 
-4. (Optional) If you're using Grafana with InfluxDB, then you can import [example Dashboard](static/dashboard.json) from first screen.
+## Deployment
+
+In order to deploy your worker, you need to have a [Cloudflare API token](https://developers.cloudflare.com/workers/wrangler/migration/v1-to-v2/wrangler-legacy/authentication/#api-token) and running instance of InfluDB.
```

Comment:
What is the reason for using the legacy v1 version of wrangler? Can we use the current version v2 instead?

https://developers.cloudflare.com/workers/wrangler/migration/v1-to-v2/

Reply:
Hey I've somehow posted the wrong link, I'm going to fix that.

Reply:
This should be correct one.